components/Caretaker.js

```diff
@@ -47,7 +47,7 @@ class Caretaker extends React.Component {
     return (
       <View>
         <Header
-          containerStyle={{ backgroundColor: 'black', justifyContent: 'space-between' }}
+          containerStyle={{ backgroundColor: '#483d8b', justifyContent: 'space-between' }}
```

Comment:
I know we mentioned it, but it might not have ever gotten into an issue. I think we could pull these objects out into their own StyleSheet, and just reference them here. It would be a small refactor, but just making a note of it again.